tutorials/qdrant_langchain_instrumentation_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,6089 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    def evaluate_query_and_retrieved_context(query: str, context: str, model_name: str) -&gt; str:
</pre><p>we can use LLM evals now for this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2440/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>